src/SourceBuild/content/eng/Versions.props

```diff
@@ -25,6 +25,8 @@
     <PrivateSourceBuiltSdkVersion>9.0.100-alpha.1.24067.1</PrivateSourceBuiltSdkVersion>
     <PrivateSourceBuiltArtifactsVersion>9.0.100-alpha.1.24067.1</PrivateSourceBuiltArtifactsVersion>
     <PrivateSourceBuiltPrebuiltsVersion>0.1.0-9.0.100-9</PrivateSourceBuiltPrebuiltsVersion>
+    <PrivateSourceBuiltNoTargetsSdkVersion>3.7.0</PrivateSourceBuiltNoTargetsSdkVersion>
```

Comment:
Can these two new sdk versions be removed now?

Reply:
Fixed with https://github.com/dotnet/installer/pull/18478/commits/a7abafeaef29f379a952cf3df80fda4299d240f3